profiles/bicycle.lua

```diff
@@ -12,7 +12,7 @@ access_tag_restricted = { ["destination"] = true, ["delivery"] = true }
 access_tags_hierarchy = { "bicycle", "vehicle", "access" }
 cycleway_tags = {["track"]=true,["lane"]=true,["opposite"]=true,["opposite_lane"]=true,["opposite_track"]=true,["share_busway"]=true,["sharrow"]=true,["shared"]=true }
 service_tag_restricted = { ["parking_aisle"] = true }
-restriction_exception_tags = { "bicycle", "vehicle", "access" }
+restrictions = { "bicycle" }
```

Comment:
The idea is to only use restriction exceptions in the same way we are using namespaced restrictions. If you allow `restriction:motorcar` you also want to check for `except=motorcar` after all.

Before we had the situation that we were using `restriction:*=` and then were using the exception tag to filter again.

(Also the exceptions list before was wrong e.g. there is no vehicle or access exception http://taginfo.openstreetmap.org/keys/except#values, also no such restrictions http://taginfo.openstreetmap.org/keys/restriction#values).